app/controllers/admin/loans_controller.rb

```diff
@@ -55,7 +54,8 @@ def show
         prep_logs(@loan)
       when "transactions"
         # requires additional level of validation beyond approrpiate loan access
-        authorize :'accounting/transaction', :index?
+        @sample_transaction = ::Accounting::Transaction.new(project: @loan)
```

Comment:
Using a `sample` object like this when authorizing `create?` is better than using the raw class in my experience b/c it lets you set up attributes like `project`. We just have to make sure that the chosen loan is enforce on submit, which we do elsewhere.

Reply:
cool